### PR TITLE
Create sk.json

### PR DESCRIPTION
**Pull request by misa1515**

### CODE REVIEW

File: custom_components/retry/translations/sk.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
+{
+    "title": "Retry",
+    "config": {
+        "abort": {
+            "single_instance_allowed": "Already configured. Only a single configuration possible."
```
Comment (amitfin) [inline custom_components/retry/translations/sk.json:5]: This is still in Engligh.

File: custom_components/retry/translations/sk.json [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        },
+        "step": {
+            "user": {
+                "description": "Do you want to start setup?"
```
Comment (amitfin) [inline custom_components/retry/translations/sk.json:9]: Dito.

**Review by amitfin — CHANGES_REQUESTED** [review]

**Comment by misa1515** [comment]
updated

**Comment by amitfin** [comment]
Thanks @misa1515 !
PR is included in [v2.3.2](https://github.com/amitfin/retry/releases/tag/v2.3.2).

**Pull request merged**